### ISSUE TITLE
When `resp.data.job_complete` returns false, an error occur NoMethodError in bigquery/query_data.rb#L110

### ISSUE THREAD

**Issue by yhirano55** [GoogleCloudPlatform/gcloud-ruby#530]
I found a bug.

**call [] in spite of gapi["schema"] is nil**

NoMethodError:
https://github.com/GoogleCloudPlatform/gcloud-ruby/blob/master/lib/gcloud/bigquery/query_data.rb#L110

I think it might need to modify [bigquery/project#L154](https://github.com/GoogleCloudPlatform/gcloud-ruby/blob/master/lib/gcloud/bigquery/project.rb#L154)

```
if resp.success? && resp.data.job_complete
  Job.from_gapi resp.data, connection
else
  fail ApiError.from_response(resp)
end
```

**Comment by quartzmo** [comment]
Thank you @yhirano55 for reporting this issue. Do you encounter this error consistently while using the BigQuery service? Can you provide a backtrace?

**Comment by blowmage** [comment]
@yhirano55 Thanks for the bug report! I have reproduced the error in tests and fixed the bug. We are very close to a release, and this fix will be included in it. If you ever have any other problems or suggestions for improvement please open another issue!

**Comment by blowmage** [comment]
This fix included in the [0.6.2 release](https://github.com/GoogleCloudPlatform/gcloud-ruby/releases/tag/v0.6.2).
